pcweb/pages/index/demos/auth/auth.py

```diff
@@ -1,70 +1,154 @@
+                ),
+                class_name="flex flex-col gap-2",
             ),
-            rx.button(
-                "Google",
-                variant="outline",
+            rx.box(
+                rx.box(
+                    rx.text(
+                        "Password", class_name="font-medium font-small text-slate-11"
+                    ),
+                    rx.link(
+                        "Forgot password?",
+                        href="#",
+                        underline="none",
+                        class_name="font-small text-violet-9",
+                    ),
+                    class_name="flex flex-row justify-between w-full",
+                ),
+                rx.el.input(
+                    placeholder="Enter your password",
+                    name="password",
+                    type="password",
+                    required=True,
+                    class_name="box-border border-slate-5 focus:border-violet-9 focus:border-1 bg-white-1 shadow-small p-[0.5rem_0.75rem] border rounded-[10px] font-small text-slate-11 placeholder:text-slate-9 outline-none focus:outline-none w-full",
+                ),
+                class_name="flex flex-col gap-2",
             ),
-            rx.button(
-                "Facebook",
-                variant="outline",
+            button("Sign In", type="submit", class_name="-mt-2"),
+            rx.box(
+                rx.divider(margin="0", class_name="bg-slate-4"),
+                rx.text(
+                    "OR CONTINUE WITH",
+                    class_name="text-slate-9 font-small !text-xs whitespace-nowrap",
+                ),
+                rx.divider(margin="0", class_name="bg-slate-4"),
+                class_name="flex flex-row gap-2 items-center",
             ),
-            direction="column",
-            spacing="2",
-            width="100%",
-        )
+            icon_button("GitHub", "github", variant="secondary"),
+            on_submit=AuthState.submit,
+            class_name="flex flex-col gap-6 border-slate-5 bg-white-1 shadow-small p-8 border rounded-[1.125rem] w-full",
+        ),
+        class_name="flex items-center px-12 py-8 h-full overflow-hidden",
+    )
 
-def auth_form():
-    return rx.fragment(
-        rx.box(
-            rx.card(
-                rx.flex(
-                    rx.text(
-                        "Create an Account",
-                        weight="bold",
-                        size="3"
-                    ),
+
+auth_code = """class AuthState(rx.State):
+
+    def submit(self, form_data):
+        return rx.toast(form_data)
+
+def auth() -> rx.Component:
+    return rx.card(
+        rx.form(
+            rx.vstack(
+                rx.image(
+                    src="/logo.jpg",
+                    class_name="image",
```

Comment:
Why do we need class names? I'm worried when they copy it they're not going to get the same result + for a first time person who sees reflex they may think its required